invenio_files_rest/models.py

```diff
@@ -114,6 +114,27 @@ def as_bucket_id(value):
     return value.id if isinstance(value, Bucket) else value
 
 
+def as_object_version(value):
```

Comment:
This method is unused as far as I can see.

Reply:
We are heavily using it in videos: https://github.com/CERNDocumentServer/cds-videos/blob/cdslabs_qa/cds/modules/deposit/api.py#L47